src/prng/isaac_array.rs

```diff
@@ -8,9 +8,60 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! ISAAC serde helper functions.
+//! ISAAC helper functions for 256-element arrays.
 
-pub(super) mod rand_size_serde {
+// Terrible workaround because arrays with more than 32 elements do not
+// implement `AsRef`, `Default`, `Serialize`, `Deserialize`, or any other
+// traits for that matter.
+
+#[cfg(feature="serde-1")] use serde::{Serialize, Deserialize};
+
+const RAND_SIZE_LEN: usize = 8;
+const RAND_SIZE: usize = 1 << RAND_SIZE_LEN;
+
+
+#[derive(Copy, Clone)]
+#[allow(missing_debug_implementations)]
+#[cfg_attr(feature="serde-1", derive(Serialize, Deserialize))]
+pub struct IsaacArray<T> where T: Default + Copy {
```

Comment:
`Default` and `Copy` should never be used in trait bounds on a data structure -- only on impls and methods. The only code that requires these bounds are the Deserialize and Default impls, so the bounds should be on those impls.

Reply:
Thank you for reviewing, I hope I got them right now. It compiles...